mixer/cmd/root.go

```diff
@@ -96,9 +110,16 @@ func Execute() {
 }
 
 func init() {
+	defaultFile := ""
+	b, err := builder.NewFromConfig(configFile)
+	if err == nil {
+		defaultFile = b.Config.Mixer.LogFilePath
+	}
+
 	RootCmd.PersistentFlags().StringVar(&rootCmdFlags.cpuProfile, "cpu-profile", "", "write CPU profile to a file")
 	_ = RootCmd.PersistentFlags().MarkHidden("cpu-profile")
-
+	RootCmd.PersistentFlags().StringVar(&rootCmdFlags.logFile, "log", defaultFile, "Write logs to a file")
+	RootCmd.PersistentFlags().IntVar(&rootCmdFlags.logLevel, "logLevel", 4, "Set the log level between 1-5")
```

Comment:
I noticed that the Mixer flags have the syntax --\<term1\>-\<term2\>.
So lets rename the `--logLevel` flag to `--log-level`.